### ISSUE TITLE
Not all VCNL4040_LEDCurrent values set correctly

### ISSUE THREAD

**Issue by ChuckVanzant** [adafruit/Adafruit_VCNL4040#5]
I noticed the Proximity LED Current was not being set to particular values.

Stepping through all the enumerated values of type VCNL4040_LEDCurrent, I see the following values returned for the values set:
Value Set --> Value Returned
- VCNL4040_LED_CURRENT_50MA    -->    VCNL4040_LED_CURRENT_50MA  
- VCNL4040_LED_CURRENT_75MA    -->    VCNL4040_LED_CURRENT_75MA  
- VCNL4040_LED_CURRENT_100MA  -->    VCNL4040_LED_CURRENT_100MA  
- VCNL4040_LED_CURRENT_120MA  -->    VCNL4040_LED_CURRENT_120MA  
- VCNL4040_LED_CURRENT_140MA  -->    **VCNL4040_LED_CURRENT_50MA**  
- VCNL4040_LED_CURRENT_160MA  -->    **VCNL4040_LED_CURRENT_75MA**  
- VCNL4040_LED_CURRENT_180MA  -->    **VCNL4040_LED_CURRENT_100MA**  
- VCNL4040_LED_CURRENT_200MA  -->    **VCNL4040_LED_CURRENT_120MA**  

The code for testing all the configuration values is attached in a zip file.
PlatformIO was used for development using library adafruit/Adafruit VCNL4040@^1.0.1
[main.zip](https://github.com/adafruit/Adafruit_VCNL4040/files/8189261/main.zip)
`

**Comment by ladyada** [comment]
thanks, please submit a PR for any changes or fixes!

**Comment by ChuckVanzant** [comment]
It looks like the number of bits that should be written to and read from for the setProximityLEDCurrent and getProximityLEDCurrent functions should be 3 instead of 2
per page 11, Table 9 of the datasheet:
```
LED_I 2 : 0
(0 : 0 : 0) = 50 mA; (0 : 0 : 1) = 75 mA; (0 : 1 : 0) = 100 mA; (0 : 1 : 1) = 120 mA
(1 : 0 : 0) = 140 mA; (1 : 0 : 1) = 160 mA; (1 : 1 : 0) = 180 mA; (1 : 1 : 1) = 200 mA
LED current selection setting
```
and a preliminary test suggests that corrects the problem.

**Comment by ChuckVanzant** [comment]
Issue was resolved by PR #6.
